_posts/2020-04-17-snowplow-ios-tracker-1.3.0-released/index.md

```diff
@@ -0,0 +1,309 @@
+    ...
+    [builder setGdprContextWithBasis:SPGdprProcessingBasisConsent
+                          documentId:@"someId"
+                     documentVersion:@"0.1.0"
+                 documentDescription:@"a demo document description"];
+}];
+{% endhighlight %}
+
+<h2 id="spm">3. Swift Package Manager support</h2>
+
+The Swift Package Manager is the official Apple dependency manager designed for Swift libraries. Since Xcode 11, SPM is compatible with the iOS build system.
+Thanks to the important contribution from [@vauxall][contributor], we have introduced the support of Swift Package Manager for our iOS tracker ([#474][474]). At the moment it supports only the iOS platform due to some limitations that we will resolve in future releases, but it is a great option for adding our mobile tracker and a great contribution from our active community.
+
+To install Snowplow Tracker with SPM:
+1. In Xcode, select File > Swift Packages > Add Package Dependency
+2. Add the repository of: https://github.com/snowplow/snowplow-objc-tracker
+3. Add the modules to your project's targets in the General tab > _Frameworks, Libraries & Embedded Content_ section
+ 
+<h2 id="deprecated">4. Deprecated methods</h2>
+
+In this release we've marked a few methods as deprecated:
+
+- SPEvent - `eventId`: This shouldn't be used as it can cause ambiguity about event duplication. This property will be set only by the tracker. If your code sets its own eventId we suggest to record your custom event ID in a custom context associated to all the events.
+
+- SPEvent - `timestamp`: In a future major version we will remove the ability to override the timestamp of the event. That will be set only by the tracker. A custom timestamp can be eventually added using a different method or parameter in a future release.
+
+- SPEvent - `addDefaultParamsToPayload:`: This shouldn't be used as it's an internal method used by the tracker and it can be subject of future changes. 
+
+Other methods have also been marked as deprecated because they are related to internal logic of the tracker. We strongly suggest avoiding the use of these methods marked deprecated as they will be substituted or removed in the next major version.
+
+
+<h2 id="updates">5. Updates and bug fixes</h2>
+
+- Fixed some warnings due to deprecated methods on last iOS versions ([#493][493], [#492][492], [#479][479]).
+
+- Internal events refactoring ([#489][489]).
+
+<h2 id="documentation">6. Documentation</h2>
+
+As always, information about how to use the tracker can be found in the [iOS Tracker documentation][docs].
+
+You can find the full release notes on GitHub as [Snowplow iOS Tracker v1.3.0 release][1.3.0-tag].
+
+
+<h2 id="help">7. Getting help</h2>
+
+For help on integrating the tracker please have a look at the setup guide. If you have any questions or run into any problems, please visit our [Discourse forum][forums]. Please raise any bugs in the [iOS Tracker’s issues][issues] on GitHub.
```

Comment:
What is the process for open source vs. insights customers integrating the tracker? If they are different maybe we can add a sentence about that here.

Reply:
Honestly I don't know. Everyone can raise a bug on GitHub. I guess Insight customers can contact us with Zendesk in case of issues. Actually I've copied and pasted this section by previous blog-post and from what I've seen no one mention insights. I can update it with anything better you suggest.